crates/uv/src/commands/reporters.rs

```
@@ -95,19 +104,27 @@ impl ProgressReporter {
         );
 
         progress.set_style(ProgressStyle::with_template("{wide_msg}").unwrap());
-        progress.set_message(format!(
-            "{} {}",
+        let message = format!(
+            "   {} {}",
             "Building".bold().cyan(),
             source.to_color_string()
-        ));
+        );
+        if multi_progress.is_hidden() && !*HAS_UV_TEST_NO_CLI_PROGRESS {
+            let _ = writeln!(self.printer.stderr(), "{message}");
```

We could raise the errors here if we need.